src/devices/Mcp23xxx/tests/Mcp23xxx.Tests.csproj

```diff
@@ -10,7 +10,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
     <PackageReference Include="xunit" Version="2.4.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
-    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" />
+    <ProjectReference Include="..\..\..\System.Device.Gpio\System.Device.Gpio.csproj" />
```

Comment:
You'll need to specify the RuntimeIdentifier in here as an additional property as well temporary

Reply:
Fixing.